scss/layout.scss

```diff
@@ -0,0 +1,10 @@
+.layout {
+  display: flex;
+  flex-flow: row wrap;
+  margin: 30px 70px 100px 70px;
+  width: 890px;
```

Comment:
fixed pixel widths tend to break the layout, use:
```
width: 100%;
max-width: 890px;
```
so the layout shrinks in smaller screens.